inspirehep/modules/workflows/models.py

```diff
@@ -74,3 +77,26 @@ class WorkflowsPendingRecord(db.Model):
         nullable=False,
     )
     record_id = db.Column(db.Integer, nullable=False)
+
+
+class WorkflowsRecordSources(db.Model):
+
+    __tablename__ = "workflows_record_sources"
+    __table_args__ = (
+        db.PrimaryKeyConstraint('record_id', 'source'),
+    )
+    source = db.Column(db.Text, default="", nullable=False)
+    record_id = db.Column(
+        UUIDType,
+        db.ForeignKey("records_metadata.id", ondelete='CASCADE'),
```

Comment:
I think you should not check this PR anymore, since it's superseded by #2423 where they have implemented all this business logic.

There is no record in this table at the beginning. This table is populated the first time only after the record is actually accepted by the curator and added into the system (thus receiving a UUID).